### PR TITLE
Add ConvertTime Class with Time Conversion Functionality and Unit Tests

### PR DESCRIPTION
**Pull request by arafat-web**
**Description:**  
This pull request introduces the `ConvertTime` class, providing a robust time conversion function alongside a complete suite of unit tests to ensure accurate and reliable operation. 

### Key Features:
- **`convertTime` Function**: Converts a specified value between various time units (e.g., seconds, minutes, hours, days, weeks, months, and years) using predefined conversion factors.
- **Exception Handling**: Throws clear exceptions for invalid input units, enhancing user feedback and preventing errors.
- **PSR-12 Compliance**: The code adheres to PSR-12 standards, ensuring readability and maintainability.

### Unit Tests:
The `ConvertTimeTest` class covers the following:
- **Basic Conversion Tests**: Verifies correct conversions between time units.
- **Edge Case Handling**: Tests zero-value conversions for non-negative results.
- **Invalid Unit Handling**: Tests exception handling for invalid `from` and `to` units with descriptive error messages.

This addition enhances functionality by enabling easy and accurate time conversions while ensuring reliability through comprehensive testing.

### CODE REVIEW

File: Conversions/ConvertTime.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    return round($result, 2);
+}
+
+// Example usage
+try {
+    $timeResult = convertTime(120, 'minutes', 'seconds');
+    echo "120 minutes = $timeResult seconds\n";
+} catch (InvalidArgumentException $e) {
+    echo "Error: " . $e->getMessage() . "\n";
+}
```
Comment (darwinz) [inline Conversions/ConvertTime.php:91]: Let's keep examples like this in the test code, with assertions

Comment (arafat-web) [reply Conversions/ConvertTime.php:91]: 4f81e1528233a1c8a6f6c96ba79ed587f7873268

Comment (darwinz) [inline Conversions/ConvertTime.php:91]: Please remove these 7 lines of code from ConvertTime.php
